demo/vl-pager.html

```diff
@@ -46,15 +46,15 @@ <h2>Pager zonder pagina items</h2>
       </code>
     </pre>
     <div class="demo">
-      <vl-pager align-center total-items="100" items-per-page="10" current-page="1" pagination-disabled></vl-pager>
+      <vl-pager id="pager-no-pagination" align-center total-items="100" items-per-page="10" current-page="1" pagination-disabled></vl-pager>
```

Comment:
misschien al vijgen na pasen maar waarom zou ge in ne page pagination-disabled nodig hebben? zet dan gene pager?

Reply:
Moet ge eens vragen aan Tom, deze PR is enkel voor de testen.